Chap 1/Chapter 1 CS/chapter1_1_home.txt

```diff
@@ -123,7 +123,10 @@ Swallowing hard, you consider the tactics you could employ as the reliable ack-a
 		At once, you pick up your rifle, sprinting across the treacherous killing ground. Less than 10 yards to go! The dreaded ack-ack of the guns erupt again, a few bullets straying near you as you dive into the dirt, slamming onto the ground! In a flash, you're pressed against the wall, keeping your figure compact as you peer upwards to judge the next move.
 		*line_break
 		*line_break
-		"FUCK THEY GOT HIM!!!" You turn, seeing one of the men standing still, his body wobbling slightly as his face contorts in shock, blood pouring down his uniform. His rifle slides out of his hand, clattering to the ground. [b]Bang! Bang!![/b] A puff of red mist explodes in his chest, hurling his body onto the ground like a rag-doll.
+		"FUCK THEY GOT HIM!!!" You turn, seeing one of the men standing still, his body wobbling slightly as his face contorts in shock, blood pouring down his uniform. His rifle slides out of his hand, clattering to the ground. [b]Bang! Bang!![/b] A puff of red mist explodes in his chest, hurling his body onto the ground like a rag-doll. 
+		*line_break
+		*line_break
+		His buddy, the person who yelled, turned around to try and help his friend up, but with his back turned he didn't see the second round of bullets until they entered his back and came out of his chest.
```

Comment:
His buddy, the person who yelled, turned around to try and help his friend up, but with his back turned he didn't see the second round of bullets until they entered his back and came out of his chest.

This seems a bit awkward - instead of being so explicit, try to describe the scene. For example:

"Don't die on me you bastard!!" One of the men tries to pull the dead soldier to his feet, throwing the corpse over his shoulder.

[b]Bang!![/b]

The man freezes, his face contorted in shock as he crashes to the ground, holes in his chest.